docs/explanations/architecture/styles.md

```diff
@@ -1,23 +1,28 @@
-## Styles in the block editor
+## Styles in the editor
 
 This document introduces the main concepts related to styles that affect the user content in the block editor. It points to the relevant reference guides and tutorials for readers to dig deeper into each one of the ideas presented. It's aimed to block authors and people working in the block editor project.
 
 1. [HTML and CSS](#html-and-css)
 2. [Block styles](#block-styles)
-  - [From UI controls to HTML markup](#from-ui-controls-to-html-markup)
-  - [Block Supports API](#block-supports-api)
-  - [Current limitations of the Block Supports API](#current-limitations-of-the-block-supports-api)
+
+-   [From UI controls to HTML markup](#from-ui-controls-to-html-markup)
+-   [Block Supports API](#block-supports-api)
+-   [Current limitations of the Block Supports API](#current-limitations-of-the-block-supports-api)
+
 3. [Global styles](#global-styles)
```

Comment:
Although we are trying to remove the `Global` from the `Global Styles` high-level concept and UI, this page dives more into implementation details, where the `global` is still valid and important to follow.

@oandregal would you mind updating this file as you see fit when you have some time?

Reply:
Sure, happy to. I'd need some direction as to how to update it.

My understanding is that the difference between block-level styles (serialized to post content as part of the block) and global-level styles (part of a stylesheet that's generated dynamically) is still relevant.